src/masternodes/loan.h

```diff
@@ -52,6 +52,67 @@ struct CLoanSetCollateralTokenMessage : public CLoanSetCollateralToken {
     }
 };
 
+class CLoanSetLoanToken
+{
+public:
+    std::string symbol;
+    std::string name;
+    uint256 priceFeedTxid;
+    bool mintable = false;
+    CAmount interest = 0;
+    DCT_ID tokenId;
+
+    ADD_SERIALIZE_METHODS;
+
+    template <typename Stream, typename Operation>
+    inline void SerializationOp(Stream& s, Operation ser_action) {
+        READWRITE(symbol);
+        READWRITE(name);
+        READWRITE(priceFeedTxid);
+        READWRITE(mintable);
+        READWRITE(interest);
```

Comment:
`tokenId` isn't serialized.

Reply:
tokenId is a leftover from some previous implementation. Will remove it.